README.md

```diff
@@ -38,10 +38,25 @@ This template comes with:
 
 ## How to use with docker
 
-1. Have docker and docker-compose installed
-1. build and run the application with `docker-compose up`
-1. Run `docker-compose exec web bash -c "rails db:create db:migrate"`
-1. You can now try your REST services!
+1. Have `docker` and `docker-compose` installed (You can check this by doing `docker -v` and `docker-compose -v`)
+2. Modify the following lines in the `database.yml` file:
```

Comment:
you can leave `1.` and it will take the correct number automatically

Reply:
If I leave it with `1.`, the numbering restarts after the code block :eyes: